src/docfx/build/Builder.cs

```diff
@@ -30,6 +30,13 @@ public static bool Run(CommandLineOptions options, Package? package = null)
             var operation = Telemetry.StartOperation("build");
             using var errors = new ErrorWriter(options.Log);
 
+            if (options.Continue)
+            {
+                // Apply templates.
+                ApplyTemplates.Run(errors, options);
+                return errors.HasError;
```

Comment:
Should also call line 46/48, consider not return here.

Reply:
Fixed.